asm/f32/sdot.go

```diff
@@ -4,16 +4,16 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package asm
+package f32
 
-func DsdotUnitary(x, y []float32) (sum float64) {
+func SdotUnitary(x, y []float32) (sum float64) {
```

Comment:
Do we want `Dsdot` or `Ddot`, since the full signature is going to be `f32.Ddot`?